client/data/sites/use-site-excerpts-query.ts

```diff
@@ -44,7 +44,8 @@ export const useSiteExcerptsQuery = (
 	sitesFilterFn?: ( site: SiteExcerptData ) => boolean,
 	site_visibility: SiteVisibility = 'all',
 	additional_fields: string[] = [],
-	additional_options: string[] = []
+	additional_options: string[] = [],
+	enabled = true
```

Comment:
Having a flag to disable the query based on the viewport size seems suboptimal and a poor design choice. I'd very much prefer we don't call the hook at all if we don't need this data?